README.md

```diff
@@ -1,47 +1,61 @@
-# GoodData.UI SDK
-
-## Getting Started
-
-The easiest way to start developing analytical applications using GoodData.UI SDK is to use
-the [Accelerator Toolkit](https://github.com/gooddata/gooddata-create-gooddata-react-app). You will
-be up and running in minutes.
-
-For detailed description of available components and capabilities see the [official documentation](https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html).
-
-You can also view our [live examples](https://gdui-examples.herokuapp.com/login) or start the live examples
-[application locally](examples/sdk-examples).
+<p align="center">
+  <img src="assets/gooddata-logo.svg" alt="GoodData Logo" width="120px" height="120px"/>
+</p>
+<h1 align="center">GoodData.UI</h1>
+
+<p align="center">
+  <a href="https://sdk.gooddata.com/gooddata-ui"><strong>sdk.gooddata.com/gooddata-ui</strong></a>
+  <br>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/@gooddata/sdk-model">
+        <img src="https://img.shields.io/npm/v/@gooddata/sdk-model.svg?logo=npm" alt="GoodData.UI on npm">
+    </a>&nbsp;
+    <a href="https://npmcharts.com/compare/@gooddata/sdk-model?minimal=true">
+        <img src="https://img.shields.io/npm/dm/@gooddata/sdk-model.svg" alt="GoodData.UI on npmcharts">
+    </a>&nbsp;
+    <img src="https://img.shields.io/badge/typescript-first-blue?logo=typescript" alt="GoodData TypeScript support badge">
+    &nbsp;
+    <a href="https://www.gooddata.com/slack">
+        <img src="https://img.shields.io/badge/slack-community-green?logo=slack" alt="GoodData Community Slack">
+    </a>&nbsp;
+</p>
+
+[GoodData.UI](https://sdk.gooddata.com/gooddata-ui) is a TypeScript framework for building analytical applications on top of the [GoodData platform](https://help.gooddata.com/doc/enterprise/en), [GoodData Cloud](https://www.gooddata.com/developers/cloud-native/doc/cloud/) or [GoodData.CN](https://www.gooddata.com/developers/cloud-native/doc/cloud/).
```

Comment:
@no23reason a little weird that both links are going to the same docs. Perhaps we could combine them?
```
[GoodData Cloud or GoodData.CN](https://www.gooddata.com/developers/cloud-native/doc/cloud/)
```